library/spdm_responder_lib/libspdm_rsp_psk_exchange.c

```diff
@@ -359,6 +360,11 @@ return_status spdm_get_response_psk_exchange(IN void *context,
 
         spdm_set_session_state(spdm_context, session_id,
                        LIBSPDM_SESSION_STATE_ESTABLISHED);
+        result = start_watchdog(&spdm_context->watchdog_context, session_id,
+            spdm_context->local_context.heartbeat_period);
```

Comment:
`local_context.heartbeat_period` is in units of seconds and `start_watchdog` uses units of microseconds.